lib/LINE/Bot/Audience.pm

```diff
@@ -45,6 +47,21 @@ sub create_audience_for_uploading {
     LINE::Bot::API::Response::AudienceGroup->new(%{ $res });
 }
 
+sub get_data_for_multiple_audience {
```

Comment:
IMHO this subroutine should be reimplement by using `URI::QueryParam` instead, for the benefit of escaping all parameter values -- imagine what happens if `$opts->{page}` contains value like `'=1&size=1000000#`.